File: BusinessLogic/Models/BranchDto.cs

namespace BusinessLogic.Models;

/// <summary>
/// Tabla de sucursales del sistema
/// </summary>
public partial class BranchDto
{
    /// <summary>
    /// Identificador único de la sucursal
    /// </summary>
    public string BranchId { get; set; } = null!;

    /// <summary>
    /// Nombre de la sucursal
    /// </summary>
    public string Name { get; set; } = null!;

    /// <summary>
    /// Dirección de la sucursal
    /// </summary>
    public string? Address { get; set; }

    /// <summary>
    /// Número de teléfono de la sucursal
    /// </summary>
    public string? Phone { get; set; }

    public IEnumerable<BranchProductDto> ProductsInBranch { get; set; } = null!;
}

File: Presentation/Angular/src/app/temp/models/branch-dto.ts
/* tslint:disable */
/* eslint-disable */
import { BranchProductDto } from './branch-product-dto';
export interface BranchDto {
  address?: null | string;
  branchId?: null | string;
  name?: null | string;
  phone?: null | string;
  productsInBranch?: null | Array<BranchProductDto>;
}
